nf_core/modules/lint/module_tests.py

```diff
@@ -52,11 +52,31 @@ def module_tests(module_lint_object, module):
                     if not tag in [module.module_name, module.module_name.split("/")[0]]:
                         all_tags_correct = False
 
+                # Look for md5sums of empty files
```

Comment:
Is it a common use case to have many identical files or would it make sense to default to not allowing multiple files with the same checksum?

Reply:
I can't think of any cases when a single module would produce multiple identical files, no. But I suspect it's (a) rare enough to not worry about and (b) inevitable that there will be an edge case exception if we do implement this 😅

Reply:
To answer that question - yes there are a lot of instances where md5sums are shared: https://github.com/nf-core/modules/pull/1167 (that screenshot is capped at being shared 5 times or more, but it's a long tail down to 2..)

Reply:
Yes, but I think that's the answer to another question (also well worth asking :)

What I was fishing for here is whatever there are good reasons for anything tested (modules) to have multiple identical outputs (which is my understanding of what it would mean to have several items with the same checksum here).

(If possible, I believe avoiding identical outputs would be good as they consume disk space and resources for moving around. But there could of course be good reasons I don't see for why it makes sense for something to have several identical outputs.)

Reply:
ok, so a single module creating multiple files with identical (non-zero) content? Seems unlikely but yeah I have no idea 🤔